src/internet_identity/Cargo.toml

```diff
@@ -53,6 +49,7 @@ canister_tests.workspace = true
 hex-literal = "0.4"
 regex.workspace = true
 ic-response-verification.workspace = true
+identity_jose = { git = "https://github.com/frederikrothenberger/identity.rs.git", branch = "frederik/wasm-test", default-features = false, features = ["iccs"]}
```

Comment:
Ideally, this would be removed too. Is the infrastructure provided by `vc_util` not good enough for the tests?

Reply:
`vc_util` doesn't have currently the corresponding functionality.  I'll add it in a follow-up PR.